### PR TITLE
Relationships

### PR DESCRIPTION
**Pull request by inspecta**
- Created vets table
- Create specialization and visits join tables
- Inserted data into vets, specialization and visits tables
- Wrote and queried SQL statements
- Took screenshots of the SQL statement results

**Screenshots of the results of the Queries **

Screenshot from 2022-12-08 18-48-15

Screenshot from 2022-12-08 18-48-43

Screenshot from 2022-12-08 18-49-43

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -55,3 +54,149 @@ WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
...
+INSERT INTO specializations (species_id, vet_id) VALUES( 
+  (SELECT id FROM species WHERE name = 'Digimon'),
+  (SELECT id FROM vets WHERE name = 'Stephaine Mendez')
+);
+
+INSERT INTO specializations (species_id, vet_id) VALUES( 
+  (SELECT id FROM species WHERE name = 'Digimon'),
+  (SELECT id FROM vets WHERE name = 'Jack Harkness')
+);
+
```
Comment (thecodechaser) [inline data.sql]: - [ ] You missed adding the following data to `specialization`, Please make sure to add that.
  - Vet Stephanie Mendez is specialized in Pokemon.

File: data.sql [hunk 2]
```diff
@@ -55,3 +54,149 @@ WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
...
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES(
+(SELECT id FROM animals WHERE name = 'Pikachu'),
+(SELECT id FROM vets WHERE name = 'Maisy Smith'),
+'05/01/2020');
+
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES(
+(SELECT id FROM animals WHERE name = 'Pikachu'),
+(SELECT id FROM vets WHERE name = 'Maisy Smith'),
```
Comment (thecodechaser) [inline data.sql]: - [ ] You missed adding the following data to `visits`, Please make sure to add that.
  - Agumon visited William Tatcher on May 24th, 2020.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @inspecta👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @inspecta 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**